libs/payments/paypal/src/lib/paypal.repository.ts

```diff
@@ -0,0 +1,87 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import {
+  AccountDatabase,
+  PaypalCustomerUpdate,
+  NewPaypalCustomer,
+} from '@fxa/shared/db/mysql/account';
+
+import { PaypalCustomerNotUpdatedError } from './paypal.error';
+import { ResultPaypalCustomer } from './types';
+
+/**
+ * Creates a paypalCustomer in the database.
+ *
+ * @returns The created paypalCustomer or throws an error if the paypalCustomer if it couldn't be created
+ */
+export async function createPaypalCustomer(
+  db: AccountDatabase,
+  paypalCustomer: NewPaypalCustomer
+) {
+  await db
+    .insertInto('paypalCustomers')
+    .values(paypalCustomer)
+    .executeTakeFirstOrThrow();
+  return fetchPaypalCustomerByUid(db, paypalCustomer.uid);
+}
+
+/**
+ * Fetch a paypalCustomer from the database by uid.
+ *
+ * @returns Fetched paypalCustomer or throws an error if the paypalCustomer does not exist
+ */
+export async function fetchPaypalCustomerByUid(
```

Comment:
`models/auth/index.ts > getAllPayPalBAyUid()` sorts results by `createdAt` in descending order. Should the same be done here?

Reply:
Shouldn't this only ever find at max one result? I assume uid is a unique key... right?